### DISCUSSION TITLE
Create a guide on how to take down stolen copyrighted content

### DISCUSSION THREAD

**Discussion by bilde2910** [VictiniX888/Illegal-Mod-Sites#386] [Ideas & Feedback]
A guide needs to be created to guide modders in taking down copyrighted content from various sources. The creation of this guide was initially suggested in #108.

**Comment by ryantheleach** [comment]
Seriously this is frustrating, I heard of the movement, went to the page to find where to report it to... and nothing.

**Comment by Flipp3rrr** [comment]
> Seriously this is frustrating, I heard of the movement, went to the page to find where to report it to... and nothing.
@ryantheleach 

You can report sites here under "Issues" if that's what you meant.

**Comment by berrysauce** [comment]
Might be coming soon. We'll see! ;)

**Comment by berrysauce** [comment]
Partially available at https://docs.stopmodreposts.org/guides/takedown-using-dmca
